src/bin/add/args.rs

```diff
@@ -1,4 +1,4 @@
-///! Handle `cargo add` arguments
+/// ! Handle `cargo add` arguments
```

Comment:
Oh, this is an actual error that rustfmt revealed: either use `//!` for module docs (two slashes), or `///` for item docs. (I think your were going for module docs.)